shared/cluster/src/main/java/io/pravega/common/cluster/zkImpl/ClusterZKImpl.java

```diff
@@ -64,6 +65,9 @@
 
     private final CuratorFramework client;
 
+    @Getter
+    private boolean isZKConnected = true;
```

Comment:
private final AtomicBoolean

Reply:
At line no:82, we are updating this variable to newState.isConnected() which is of type boolean.

Reply:
Modified to AtomicBoolean